.ci/mellanox/azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
```

Comment:
@artemry-mlnx - no need trigger on "merge" event into branches. only on PR.

Reply:
Fixed in https://github.com/open-mpi/ompi/pull/7465